autoscheduler/frontend/src/components/NavBar.tsx

```diff
@@ -16,7 +18,7 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const NavBar: React.SFC = function App() {
-  const classes = useStyles(useStyles);
+  const classes = useStyles(appTheme);
```

Comment:
This is now using the theme defined in `theme.ts`, so if we make changes to that theme later, they'll be passed down to the NavBar